_includes/sidebar.html

```diff
@@ -64,9 +65,9 @@ <h2>Get in touch</h2>
     </section>
 
     <!-- Footer -->
-    <footer id="footer">
+    <!-- <footer id="footer">
```

Comment:
Why is this being commented out?

Reply:
Will uncomment
The link to Andrew Branchi website is broken though